src/OrbitGl/Track.h

```diff
@@ -102,13 +101,12 @@ class Track : public orbit_gl::CaptureViewElement, public std::enable_shared_fro
 
   [[nodiscard]] virtual std::vector<CaptureViewElement*> GetVisibleChildren() { return {}; }
   [[nodiscard]] virtual int GetVisiblePrimitiveCount() const { return 0; }
-  [[nodiscard]] virtual uint32_t GetIndent() const { return indentation_level_; }
 
  protected:
   // Returns the y-position of the triangle.
   float DrawCollapsingTriangle(Batcher& batcher, TextRenderer& text_renderer,
                                uint64_t current_mouse_time_ns, PickingMode picking_mode,
-                               float z_offset = 0);
+                               float z_offset, int indentation_level);
```

Comment:
Maybe keep using `uint32_t` type for the `indentation_level` and put it before `z_offset` to keep consistent with other draw methods.

Namely, `uint32_t indentation_level, float z_offset` here.

Reply:
Thanks for catching this! I will fix it after rebasing.